javascript/adonisjs6-http/config.yaml

```diff
@@ -0,0 +1,3 @@
+framework:
+  website: adonisjs.com
+  version: 6.2
```

Comment:
Who can I ensure that this implementation use v6.2, I mean I can not find any 6.2  version in package.json

Reply:
Correct! We are in fact not using the core here 😄 
How could I represent that in the config file?

Reply:
You can't. The idea is to reflect which version of adonijsis used (not depending of module).

If I understand correctly, `http-server` is the main module here. Is their a relation between its version and adonisjs version ?

I mean `http-server` is 7.0.x, does it mean that this minor will use `adonijs` 6.2.x, like `http-server` is 7.1.x will use `adonisjs` 6.3.x

Reply:
> Is their a relation between its version and adonisjs version ?

Not really; since it is an internal dependency of `@adonisjs/core,` it is not "displayed" to the final user.
Maybe we can list it under `adonis-http-server-7.0`?

Reply:
Ok for me

Reply:
Just `adonisjs-http` will be enough, and version 7 in config file (will be displayed in UI)